src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -855,6 +859,10 @@ public Long getValue()
         rowIndexSizeAborts = createTableMeter("RowIndexSizeAborts", cfs.keyspace.metric.rowIndexSizeAborts);
         rowIndexSize = createTableHistogram("RowIndexSize", cfs.keyspace.metric.rowIndexSize, false);
 
+        tooManyIndexReadSSTablesWarnings = createTableMeter("TooManyIndexReadSSTablesWarnings", cfs.keyspace.metric.tooManyIndexReadSSTablesWarnings);
+        tooManyIndexReadSSTablesAborts = createTableMeter("TooManyIndexReadSSTablesAborts", cfs.keyspace.metric.tooManyIndexReadSSTablesAborts);
+        tooManyIndexReadSSTables = createTableHistogram("TooManyIndexReadSSTables", cfs.keyspace.metric.tooManyIndexReadSSTables, false);
```

Comment:
Same comments as the keyspace metrics in terms of naming and whether we want to move the SSTables hit histogram.